<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="product_list">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>311</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Price</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Size</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Colour</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Low</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>High</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="filter">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProducts">
    <property name="title">
     <string>Products</string>
    </property>
   </widget>
   <addaction name="menuProducts"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
